# Time for subscribing the camera topic
wait_time: 100.0 

# Max number of faces (frames with face) saved in the __frame_list
max_face_num: 30.0 

# Number of representations for one face
num_representation: 1.0  

# Pretrained model name for face recognition: VGG-Face, OpenFace, DeepFace, DeepID
model_name: VGG-Face

# Metric name for face recognition: cosine, euclidean, euclidean_l2
distance_metric: cosine 

# Detector backend for face detection: opencv, ssd, dlib, mtcnn, retinaface, mediapipe
detector_backend: opencv

# Normalize the input image before feeding to model: base, raw, Facenet, Facenet2018, VGGFace, VGGFace2, ArcFace
normalization: base
